client/blocks/reader-excerpt/index.jsx

```diff
@@ -18,10 +19,12 @@ const ReaderExcerpt = ( { post, isDiscover } ) => {
 
 	return (
 		<AutoDirection>
-			<div
-				className="reader-excerpt"
-				dangerouslySetInnerHTML={ { __html: excerpt } } // eslint-disable-line react/no-danger
-			/>
+			<Emojify>
```

Comment:
Emojify wraps it's children in a `span`, so this would result in a `div` nested inside a `span`, which isn't valid HTML5. However, it seems to work... 

Perhaps we could alter Emojify to take the element to be used for the wrapper? And then have InlineEmojify, which uses a span, and BlockEmojify, which uses a div?